.eslintrc

```diff
@@ -2,11 +2,28 @@
   "parser": "babel-eslint",
   "plugins": [
     "async-await",
-    "flowtype"
+    "flowtype",
+    "import"
   ],
   "settings": {
     "flowtype": {
       "onlyFilesWithFlowAnnotation": true
+    },
+    "import/ignore": [
+      // Because of CommonJS incompatibility, we can't
+      // check for bad imports in node_modules.
+      "node_modules",
+      // Ignore non-JS imports.
+      "\\.scss$",
```

Comment:
@shubheksha we don't have any of these `.scss`, `.jpg`, `.mp4` or `.webm` imports in our sources, right?
if it is not currently needed, I would remove the lines between 16 and 20.